### PR TITLE
Postman CI

### PR DESCRIPTION
**Pull request by snissn**

### CODE REVIEW

**Comment by snissn** [comment]
Looks good to me - successful run of the CI tool here in my repo fork - https://github.com/snissn/estuary/actions/runs/3742162420/jobs/6352739664

**Comment by snissn** [comment]
Does anyone know how to do a squash merge to make the PR come through in one commit? tysm! :)

**Comment by jcace** [comment]
I think you just need to hit this button @snissn !
Screenshot 2022-12-20 at 10 11 44 AM

**Comment by snissn** [comment]
ty @jcace def magic button ftw! :)

File: tests/postman/jsoverrides/admin_invites_code_POST.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env node
```
Comment (jcace) [inline tests/postman/jsoverrides/admin_invites_code_POST.js:1]: Are all these files like this just placeholders and don't do anything yet? To be filled in later?

**Review by jcace — APPROVED** [review]
very cool, love the idea of integration testing. Looks good to me!

**Review by alvin-reyes — APPROVED** [review]
LGTM

File: tests/postman/jsoverrides/admin_invites_code_POST.js (hunk 1 above)
Comment (snissn) [reply tests/postman/jsoverrides/admin_invites_code_POST.js:1]: these are not place holders they're important. they're manual overrides for the autogen test.

**Pull request merged**